nevergrad/functions/images/core.py

```diff
@@ -221,7 +221,7 @@ def evaluation_function(self, *recommendations: ng.p.Parameter) -> float:
         output_adv = self._get_classifier_output(x)
         _, pred = torch.max(output_adv, axis=1)
         actual = int(self.label)
-        return float(pred == actual if self.targeted else pred != actual)
+        return float(pred != actual if self.targeted else pred == actual)
```

Comment:
are you confident with this change? If I remember well it was by design, @LaurentMnr95 wanted this to go up for better results.
If you do this change please update the docstring as well.

Reply:
and this does not change anything for the optimization, it only changes the evaluation loss
(but that may make a mess with the plotting tools indeed :s)

Reply:
The goal of the original PR was precisely to switch the plotting so that it's minimization.
Laurent, can you confirm ?
This is basically Laurent's PR that was never merged.

Reply:
The original PR was approved and never merged just due to difficulties for rebasing.